src/components/progress-bar/progress-bar.module.css

```diff
@@ -18,5 +18,7 @@
 	left: 0;
 	position: absolute;
 	top: 0;
-	width: var(--percent-done);
+	width: 100%;
+	transform: scaleX(var(--portion-done));
```

Comment:
👨‍🍳 😗 👌 

Transform + width 100%, smart!

Reply:
Props to @alexcarpenter 💖